src/main/java/frc/robot/subsystems/vision/AprilTagVisionIOLimelight.java

```diff
@@ -37,8 +40,9 @@ public AprilTagVisionIOLimelight(String identifier) {
 
     observationSubscriber =
         limelightTable
-            .getStringTopic("json")
-            .subscribe("", PubSubOption.keepDuplicates(true), PubSubOption.sendAll(true));
+            .getDoubleArrayTopic("botpose_wpiblue")
```

Comment:
does this emply we are always blue alliance?

Reply:
Yes for this year you want to assume blue corner is always 0,0